crawler/helpers/cherrypicker_test_data.py

```diff
@@ -0,0 +1,154 @@
+import csv
+from datetime import datetime
+import logging
+import os
+import random
+import requests
+
+
+logger = logging.getLogger(__name__)
+
+
+def make_coords_list():
+    for column in "ABCDEFGH":
+        for row in range(1, 13):
+            yield "%s%02d" % (column, row)
+
+
+WELL_COORDS = list(make_coords_list())
+PLATE_SIZE = len(WELL_COORDS)
+
+SAMPLES_FILE_HEADERS = [
+    "Root Sample ID",
+    "Viral Prep ID",
+    "RNA ID",
+    "RNA-PCR ID",
+    "Result",
+    "Date Tested",
+    "Lab ID",
+    "CH1-Target",
+    "CH1-Result",
+    "CH1-Cq",
+    "CH2-Target",
+    "CH2-Result",
+    "CH2-Cq",
+    "CH3-Target",
+    "CH3-Result",
+    "CH3-Cq",
+    "CH4-Target",
+    "CH4-Result",
+    "CH4-Cq",
+]
+
+PRINT_FILE_HEADERS = ["barcode", "text"]
+BARACODA_PREFIX = "TEST"
+
+
+def flatten(nested_list: list) -> list:
+    return [item for sublist in nested_list for item in sublist]
+
+
+def generate_baracoda_barcodes(num_required: int) -> list:
+    baracoda_url = f"http://uat.baracoda.psd.sanger.ac.uk/barcodes_group/{BARACODA_PREFIX}/new?count={num_required}"
+    response = requests.post(baracoda_url, data={})
+    response_json = response.json()
+    barcodes: list = response_json["barcodes_group"]["barcodes"]
+    return barcodes
+
+
+def create_barcodes(num_required: int) -> list:
+    # call Baracoda here and fetch a set of barcodes with the prefix we want
+    logger.info(f"Num barcodes required from Baracoda = {num_required}")
+    list_barcodes = generate_baracoda_barcodes(num_required)
+    return list_barcodes
+
+
+def create_root_sample_id(barcode: str, well_num: int) -> str:
+    return "RSID-%s%s" % (barcode, str(well_num).zfill(4))
```

Comment:
Yes the well_num is always between 1 and 96 only.  If it went above 96, the lookup of the well coordinate in this code further down in the file would fail:

```
well_coordinate = WELL_COORDS[well_index]
```